docs/sql-ref-syntax-ddl-alter-view.md

```diff
@@ -19,4 +19,78 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+The `ALTER VIEW` statement changes various auxiliary properties of a view.
+
+
+#### Rename view
+Rename the existing view. If the view name already exists in the database, an exception is thrown. This operation does 
```

Comment:
Rename -> Renames
Also maybe clarify the exception is thrown if the _new_ name already exists

Reply:
@srowen, I changed this session to the following, let me know if you are thinking something else. 
`#### Rename VIEW

Renames the existing view. If the view name already exists in the source database, an exception is thrown. This operation does not support moving the views cross databases. `